doc/contribution.md

```diff
@@ -1,9 +1,36 @@
+<img src="resources/images/Xiana.png" width="242">
+
 # Contribution
 
+- [Ticket system](#ticket-system)
+- [Coding standards](#coding-standards)
+- [Submitting a PR](#submitting-a-pr)
 - [Development dependencies](#development-dependencies)
 - [Setup](#setup)
 - [Deps](#deps)
 - [Releasing](#releasing)
+- [Generating API docs](#generating-api-docs)
+
+## Ticket system
+
+We're using GitHub [issues](https://github.com/Flexiana/framework/issues) for tracking and discussing ideas and
+requests.
+
+## Coding standards
+
+Please follow `clj-style` and `kondo` instructions. `Kibit` isn't a showstopper, but PRs are more welcomed if not
+breaking `kibit`.
+
+## Submitting a PR
+
+Before you are submitting a PR be sure:
+
+- You've updated the documentation and the [CHANGELOG](../CHANGELOG.md)
+- the PR has an issue in GitHub, with a good description
+- you have added tests
+- you provided an example project for a new feature
+- All PRs need at least two approvals
```

Comment:
This doesn't really belong in a list of things to check. Maybe just move it to it's own paragraph after the list.